ome/omero-install
# Ice36 default

Make ice 3.6 the default value
This also change the output of the "autogen" script
the recommended version is now ice 3.6

To test:
 * travis should be green
 * run ``autogen.sh``
 * Check that that before and after the commands to install ice 3.6 there are 
``#start-recommended-ice`` and ``#end-recommended-ice``
